projects/samples/devices/controllers/laser_pointer/laser_pointer.c

```diff
@@ -25,7 +25,7 @@
 #include <webots/motor.h>
 #include <webots/robot.h>
 
-#define TIME_STEP 64
+#define TIME_STEP wb_robot_get_basic_time_step() 
```

Comment:
Even if this works correctly it would be better to use the same structure as in the other worlds.
See for example:
https://github.com/cyberbotics/webots/blob/68c3a24f07e68bd364904e5efa3d80f1c59a86d2/projects/samples/devices/controllers/axial_and_tail_rotors/axial_and_tail_rotors.c#L46